src/odemis/driver/hamamatsurx.py

```diff
@@ -1157,10 +1160,10 @@ def readCommandResponse(self):
                         continue  # return to try-statement and start receiving again
 
                     if self._getReadoutCamInfo:
-                        # command parent.CamParamGet("Setup", "CameraInfo") behaves differently then all other commands.
+                        # HACK: command CamParamGet("Setup", "CameraInfo") behaves differently then all other commands.
```

Comment:
I think I made a language error here. Think it should be `...from all the other commands`

Reply:
just a very small note: different than is not always incorrect as long as you spell it with an "a" and not an "e". https://www.grammar.com/different-from-vs-different-than/